pkg/auth/authenticator/password/ldappassword/ldap.go

```diff
@@ -122,27 +109,20 @@ func (a *Authenticator) getIdentity(username, password string) (authapi.UserIden
 		nil,          // controls
 	)
 
-	glog.V(4).Infof("searching for %s", filter)
-	results, err := l.Search(searchRequest)
+	entry, err := ldaputil.QueryForUniqueEntry(a.options.ClientConfig, searchRequest)
```

Comment:
are we calling `QueryForUniqueEntry` in a loop in the sync job? we should be reusing the connection... TLS establishment is expensive